### ISSUE TITLE
Application does not play well with Windows high contrast modes

### ISSUE THREAD

**Issue by djp952** [djp952/hdhomeruntray#43]
The application needs to work acceptably well with Windows high contrast modes

**Comment by djp952** [comment]
Windows 7: High Contrast #1, High Contrast #2, High Contrast Black - Tray icon needs to switch to white theme
Windows 10: High Contrast White - Tray icon needs to switch to dark theme
Windows 11: Desert - Tray icon needs to switch to dark theme

**Comment by djp952** [comment]
Implemented via: https://github.com/djp952/hdhomeruntray/commit/d6dcf4d547f3622e36613ec0b82591166d2f7974

The tray will use SystemColors.Control, which can be checked for its brightness in High Contrast mode and choose the icon theme based on that
